src/plugins/theme/height.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Heights - customized to not produce classes with our custom spacing values
+ */
+module.exports = function () {
+  return {
+    auto: 'auto',
```

Comment:
overriding the TW defaults because the default generated a bunch of classes using our spacing steps (which in our cases don't really make sense as heights.